build.gradle

```diff
@@ -328,6 +328,10 @@ dependencyManagement {
             entry 'accessors-smart'
         }
 
+        dependencySet(group: 'org.yaml', version: '1.31') {
```

Comment:
You should add the CVE that is the reason for doing this as a comment above like the rest.

Reply:
This will be removed before merging.